pom.xml

```
@@ -476,6 +476,65 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.openrewrite.maven</groupId>
+                    <artifactId>rewrite-maven-plugin</artifactId>
+                    <version>6.12.1</version>
+                    <configuration>
+                        <exclusions>
+                            <exclusion>**Chars.java</exclusion> <!-- // @SuppressWarnings("PMD.MissingOverride") -->
+                            <exclusion>**Jep441_PatternMatchingForSwitch.java</exclusion>
+                            <exclusion>**testdata**</exclusion>
+                        </exclusions>
+                        <activeRecipes>
```

```suggestion
                        <recipes>
```
@timtebeek

is there a way we can overload the API to have the same look and feel like the `exclusions-exclusion` division?
going from `activeRecipes-recipes` to `recipes-recipes`

for having more concise/compact API, as the rewrite context is kind of clear.